runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -262,6 +262,18 @@ public void onActivityStateChange(Activity activity, int newState) {
      */
     @XWalkAPI
     public static final int RELOAD_IGNORE_CACHE = 1;
+    /**
+     * SurfaceView as default compositing surface which has a bit performance advantage.
```

Comment:
SurfaceView is the default compositing surface. It has less latency and uses less memory.